src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -24,30 +33,119 @@ For more info on Sentry CLI configuration visit the [Sentry CLI configuration do
 
 </Note>
 
-Make sure Sentry CLI is configured for your project. You can use a `.sentryclirc` file or environment variables:
-
 <Note>
 
 Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
 
 </Note>
 
-```ìni {filename:.sentryclirc}
-[auth]
-token=your-auth-token
+## Upload with Debug ID (Preferred)
 
-[defaults]
-org=___ORG_SLUG___
-project=___PROJECT_SLUG___
+Debug IDs are a new feature we have released to account for most of the problems that users were incurring in with the release-based system. Since having events not symbolicated is big problem for our users, we designed Debug IDs to provide a reliable system that would solve the problems people were incurring.
+
```

Comment:
I would avoid explaining why we built this feature because I doubt people care. It might be better to use this time to explain what debug IDs are! 

For example; summarizing the process of injecting to uploading.

Reply:
Great feedback, will do!

Reply:
This the current level of detail is fine. If you don't explain debug ids at all, the command `sentry-cli sourcemaps inject /path/to/folder` doesn't make sense. What are you injecting? Why?

It might sense to take another pass (can be in a later PR) at this and pull out all but the bare minimum explanation of debug ids into a separate doc.